packages/launchpad/src/secp256k1hdwallet.ts

```diff
@@ -259,10 +263,11 @@ export class Secp256k1HdWallet implements OfflineSigner {
     ];
   }
 
-  public async sign(signerAddress: string, signDoc: StdSignDoc): Promise<SignResponse> {
+  public async sign(signerAddress: string, signJob: AminoSignJob): Promise<AminoSignResponse> {
```

Comment:
This should support sign mode direct, right? Or is that a separate type in stargate?

If the wallet can only sign *either* legacy *or* direct/proto (which is my current understanding with the type system), then that means they will only have exactly one sign mode. And you should replace `public readonly signModes: readonly SignMode[];` with `public readonly signMode: SignMode;`